src/ansys/sherlock/core/stackup.py

```diff
@@ -461,17 +468,17 @@ def update_laminate_layer(
         project,
         cca_name,
         layer,
-        manufacturer="",
-        grade="",
-        material="",
-        thickness=0,
-        thickness_unit="",
-        construction_style="",
-        glass_construction=[],
-        fiber_material="",
-        conductor_material="",
-        conductor_percent="",
-    ):
+        manufacturer: Optional[str] = "",
+        grade: Optional[str] = "",
+        material: Optional[str] = "",
+        thickness: Optional[float] = 0,
+        thickness_unit: Optional[str] = "",
+        construction_style: Optional[str] = "",
+        glass_construction: Optional[list[tuple[str, float, float, str]]] = None,

```

Comment:
Note that the default value for `glass_construction` was change to `None`. Having the default be `[]` causes a warning and is not a good practice.